core/src/main/scala/com/spotify/featran/FeatureBuilder.scala

```diff
@@ -274,11 +274,14 @@ object FeatureBuilder {
 
 }
 
-private class JMapWrapper[K, V](val m: java.util.Map[K, V]) extends Map[K, V] {
+private final class JMapWrapper[K, V](m: java.util.Map[K, V]) extends Map[K, V] with Serializable {
```

Comment:
It's not a case class though, confusing commit message?

Reply:
@nevillelyh ah yeah updated